### ISSUE TITLE
maxRate/clientMaxRate should be able to be float?

### ISSUE THREAD

**Issue by neoelvishsiao** [krakendio/krakend-ratelimit#13]
Right now maxRate/clientMaxRate are all integers, which can only be as small as 1 request per second. Wish it can be by minute scale (should be useful for clientMaxRate).

**Comment by Alnyli07** [comment]
It works, you can give following configuration `"github.com/alnyli07/krakend-ratelimit/juju/router": {
                    "clientMaxRate": 0.183,
                    "capacity": 1,
                    "strategy": "header",
                    "key": "x-test"
                }`

**Comment by kpacha** [comment]
I think you found an inconsistency in the juju ratelimiter: the proxy package accepts floats but the router goes with integers

https://github.com/devopsfaith/krakend-ratelimit/blob/master/juju/proxy/proxy.go#L51-L54
https://github.com/devopsfaith/krakend-ratelimit/blob/master/juju/router/router.go#L34-L39

**Comment by yanivshiloah** [comment]
Is there a chance it could be added to the router level rate limiter? We do have a use case in which we want to restrict users to make x requests per minute/hour.

**Comment by kpacha** [comment]
@yanivshiloah thanks for the heads up! We totally forgot about the branch with the fix and a minor improvement

Cheers!

**Comment by yanivshiloah** [comment]
WOW Great!! thanks a lot!

**Comment by yanivshiloah** [comment]
Hey, I don't quite understand how to use this. Say I want to allow only 60 requests per hour per IP address, how can I do this?

**Comment by kpacha** [comment]
check the documentation: https://www.krakend.io/docs/endpoints/rate-limit/#client-rate-limiting-client_max_rate

TLDR; this will do. You can fine tune the capacity and customize how to extract the client ip

```
            "extra_config":{
                "qos/ratelimit/router": {
                    "client_max_rate": 0.0167,
                    "client_capacity": 60,
                    "strategy":"ip"
                }
            }
```

**Comment by yanivshiloah** [comment]
Thanks a lot, I have no idea how I missed the client_capacity part 😂
